community/backporting_critical_changes.md

```diff
@@ -0,0 +1,61 @@
+In the event that we need to backport changesets across our various [SDKs](https://github.com/octokit) we take a fairly ridged approach to where those changes get backported to.
+
+In general, **we will backport to the last major version release**.  There are exceptions:
+1. If the change is causing issues in a specific release and no others then the backport will target that release and any future ones (where appropriate)
+2. If the change is related to a Security Advisory that is causing data corruption.
+3. If the change is related to a Security Advisory where any personally identifiable information is involved.
+4. If the change is related to a Security Advisory that is causing system wide issues / network related issues.
+5. If the change is related to an issue in a language framework or Security Advisory within that language's ecosystem.
+6. If the change is in our JavaScript SDKs. In this case the patch will be applied to the latest non ESM version (see the table below).
```

Comment:
The first point here is super clear and makes a lot of sense.
The sixth point is super clear and makes sense. 

Points 2-5 don't give a policy for where we backport to. Do we backport to _all_ major versions in those cases? 2 major versions behind? Do we have a window of 12 months or 24 months of major versions?

Reply:
I think the plan is, for now, to backport all the way to the last non-esm version